cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/lineage/LineageSerializer.java

```diff
@@ -101,7 +101,7 @@ public static LineageRecord toLineageRecord(long start, long end, Lineage lineag
   }
 
   private static String makeProgramKey(ProgramId program) {
-    return Joiner.on('.').join(program.getType().getCategoryName().toLowerCase(), program.getNamespaceId(),
+    return Joiner.on('.').join(program.getType().getCategoryName().toLowerCase(), program.getNamespace(),
```

Comment:
I had a feeling we wouldn't want to use a non-``String`` object in the ``Joiner``. Everything else was a string.